e2e/test/scenarios/embedding/sdk-iframe-embedding-setup/select-embed-options.cy.spec.ts

```diff
@@ -346,11 +349,38 @@ H.describeWithSnowplow(suiteTitle, () => {
       .first()
       .should("have.css", "color", "rgb(255, 0, 0)");
 
+    cy.log("reset button should now be visible");
+    getEmbedSidebar().findByLabelText("Reset colors").should("be.visible");
+
     cy.log("snippet should be updated");
     getEmbedSidebar().findByText("Get Code").click();
 
     codeBlock().should("contain", '"theme": {');
     codeBlock().should("contain", '"colors": {');
     codeBlock().should("contain", '"brand": "#FF0000"');
+
+    cy.log("go back to embed options step");
+    getEmbedSidebar().findByText("Back").click();
+
+    cy.log("click reset button");
+    getEmbedSidebar().findByLabelText("Reset colors").click();
+
+    H.expectUnstructuredSnowplowEvent({
+      event: "embed_wizard_option_changed",
+      event_detail: "theme",
+    });
+
+    cy.log("table header should be back to default blue");
+    H.getSimpleEmbedIframeContent()
+      .findAllByTestId("cell-data")
+      .first()
+      .should("have.css", "color", "rgb(80, 158, 227)");
+
+    cy.log("reset button should be hidden again");
+    getEmbedSidebar().findByLabelText("Reset colors").should("not.exist");
+
+    cy.log("snippet should not contain theme colors");
+    getEmbedSidebar().findByText("Get Code").click();
+    codeBlock().should("not.contain", '"theme": {');
```

Comment:
If the colors are reset, we should not show the `theme: {` field in the code snippet.